src/server/project.ts

```diff
@@ -251,7 +251,7 @@ namespace ts.server {
         public readonly getCanonicalFileName: GetCanonicalFileName;
 
         /*@internal*/
-        private exportMapCache = createExportMapCache();
+        exportMapCache: ExportInfoMap | undefined;

```

Comment:
It seems weird to expose this directly and via `getCachedExportInfoMap`.  It might be cleaner to add `clearCachedExportInfoMap` instead.